browser/src/map/handler/Map.StateChanges.js

```diff
@@ -3,13 +3,18 @@
  * L.Map.StateChanges stores the state changes commands coming from core
  * LOK_CALLBACK_STATE_CHANGED callback
  */
-/* global $ app */
+/* global $ app _ */
 /*eslint no-extend-native:0*/
 L.Map.mergeOptions({
 	stateChangeHandler: true
 });
 
 L.Map.StateChangeHandler = L.Handler.extend({
```

Comment:
StateChangeHandler seems to be not a perfect place for such things.
Let's keep it doing only one thing - storing status values for a command.
We use `_UNO('command')` for getting the translated "label" for uno command, which seems to already exist and do that correctly :) can we reuse it or there is some problem?

Reply:
Let me check, I did read it. It translate with hotkeys underscore I think.